__tests__/external-config.test.ts

```diff
@@ -59,38 +55,38 @@ test('it parses options from both sources', async () => {
 
 test('in case of conflicts, the inline config is the source of truth', async () => {
   setInput('fail-on-severity', 'low')
-  setInput('config-file', './__tests__/fixtures/config-allow-sample.yml') // this will set fail-on-severity to 'critical'
+  setInput('config-file', '../__tests__/fixtures/config-allow-sample.yml') // this will set fail-on-severity to 'critical'
 
   const config = await readConfig()
   expect(config.fail_on_severity).toEqual('low')
 })
 
 test('it uses the default values when loading external files', async () => {
-  setInput('config-file', './__tests__/fixtures/no-licenses-config.yml')
+  setInput('config-file', '../__tests__/fixtures/no-licenses-config.yml')
   let config = await readConfig()
   expect(config.allow_licenses).toEqual(undefined)
   expect(config.deny_licenses).toEqual(undefined)
 
-  setInput('config-file', './__tests__/fixtures/license-config-sample.yml')
+  setInput('config-file', '../__tests__/fixtures/license-config-sample.yml')
   config = await readConfig()
   expect(config.fail_on_severity).toEqual('low')
 })
 
 test('it accepts an external configuration filename', async () => {
-  setInput('config-file', './__tests__/fixtures/no-licenses-config.yml')
+  setInput('config-file', '../__tests__/fixtures/no-licenses-config.yml')
   const config = await readConfig()
   expect(config.fail_on_severity).toEqual('critical')
 })
 
 test('it raises an error when given an unknown severity in an external config file', async () => {
-  setInput('config-file', './__tests__/fixtures/invalid-severity-config.yml')
+  setInput('config-file', '../__tests__/fixtures/invalid-severity-config.yml')
   await expect(readConfig()).rejects.toThrow()
```

Comment:
🤔 do these paths require the `../__tests__/` prefix or would a `fixtures/` prefix work too?

Reply:
I'm not entirely sure why (maybe `jest`?), but the path needs to be relative to `src/`, hence the change from `./` to `../`:

```
[FAIL] it accepts an external configuration filename

    Unable to fetch or parse config file: ENOENT: no such file or directory, open '/Users/febuiles/w/dr-action/src/fixtures/no-licenses-config.yml'
```

Reply:
cc @hmaurer @bteng22 thoughts on this? if its nothing, feel free to ignore 👍